metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala

```diff
@@ -157,6 +157,8 @@ object ClientCommands {
     "Refresh model",
     """|Notifies the client that the model has been updated and it 
        |should be refreshed (e.g. by resending code lens request)
+       |
+       |Note: Metals will favor instead [Code Lens Refresh Request](https://microsoft.github.io/language-server-protocol/specifications/specification-3-16/#codeLens_refresh) if the client has the capability.
```

Comment:
I'm not sure what others think of this, but I'd even be in favor of putting a not here that this is now deprecated and that instead you should use the code lens refresh if your editor supports it. You include the second part, but I think we can even word this more strongly to push people towards the actual refresh.

Reply:
Something like this?

**Note**: This request is deprecated and Metals will favor [Code Lens Refresh Request](https://microsoft.github.io/language-server-protocol/specifications/specification-3-16/#codeLens_refresh) if supported by the client. 
       
Notifies the client that the model has been updated and it
should be refreshed (e.g. by resending code lens request)